includes/admin/reporting/reports.php

```diff
@@ -2548,377 +2547,454 @@ function display_export_report() {
+								<?php echo EDD()->html->year_dropdown( 'start_year' ); ?>
+							</div>
+
+							<span class="edd-to-and-from--separator"><?php echo _x( '&mdash; to &mdash;', 'Date one to date two', 'easy-digital-downloads' ); ?></span>
+
+							<div class="edd-to-and-from-container">
+							<label for="end_month" class="screen-reader-text"><?php esc_html_e( 'End Month', 'easy-digital-downloads' ); ?></label>
+								<?php echo EDD()->html->month_dropdown( 'end_month' ); ?>
+							<label for="end_year" class="screen-reader-text"><?php esc_html_e( 'End Year', 'easy-digital-downloads' ); ?></label>
+								<?php echo EDD()->html->year_dropdown( 'end_year' ); ?>
+							</div>
+							<?php wp_nonce_field( 'edd_ajax_export', 'edd_ajax_export' ); ?>
+							<input type="hidden" name="edd-export-class" value="EDD_Batch_Earnings_Report_Export"/>
+							<button type="submit" class="button button-secondary"><?php esc_html_e( 'Generate CSV', 'easy-digital-downloads' ); ?></button>
+						</form>
+					</div>
+				</div>
+
+				<div class="postbox edd-export-sales-earnings">
+					<h2 class="hndle"><span><?php esc_html_e( 'Export Sales and Earnings', 'easy-digital-downloads' ); ?></span></h2>
+					<div class="inside">
+						<p><?php esc_html_e( 'Download a CSV of all sales or earnings on a day-by-day basis.', 'easy-digital-downloads' ); ?></p>
+						<form id="edd-export-sales-earnings" class="edd-export-form edd-import-export-form" method="post">
+							<div class="edd-from-to-wrapper">
+								<label for="edd-order-export-start" class="screen-reader-text"><?php esc_html_e( 'Export Sales and Earnings Start Date', 'easy-digital-downloads' ); ?></label>
 									<?php
+									echo EDD()->html->date_field(
+										array(
+											'id'          => 'edd-order-export-start',
+											'class'       => 'edd-export-start',
+											'name'        => 'start',
+											'placeholder' => _x( 'From', 'date filter', 'easy-digital-downloads' ),
+										)
+									);
+									?>
+								<label for="edd-order-export-end" class="screen-reader-text"><?php esc_html_e( 'Export Sales and Earnings End Date', 'easy-digital-downloads' ); ?></label>
+									<?php
+									echo EDD()->html->date_field(
+										array(
+											'id'          => 'edd-order-export-end',
+											'class'       => 'edd-export-end',
+											'name'        => 'end',
+											'placeholder' => _x( 'To', 'date filter', 'easy-digital-downloads' ),
+										)
+									);
+
+									?>
+							</div>
+							<label for="edd_orders_export_download" class="screen-reader-text"><?php esc_html_e( 'All Downloads or Select Download', 'easy-digital-downloads' ); ?></label>
+								<?php
+								echo EDD()->html->product_dropdown(
+									array(
+										'name'        => 'download_id',
+										'id'          => 'edd-orders-export-download',
```

Comment:
Since EDD forces this to `_` instead of `-`, I would put it back to that so that it's clear without having to refer to the `product_dropdown` (I think actually you have to dig through to `select`) method. Anything which creates a dropdown/select input, I think EDD will do this to, so we may as well set the `id` parameter to match what's output.